src/modules/missionHelper/settings.ts

```diff
@@ -115,11 +124,15 @@ export default (MODULE_ID: string): unknown => ({
               },
           }
         : null),
-    'patients.patient_allow_first_responder_chance': {
-        type: 'toggle',
-        default: true,
-        dependsOn: '.patients.content',
-    },
+    ...(BUILD_LANG !== 'nl_NL'
```

Comment:
Why First Responder only nl_NL? The chance exists in every country and can be "triggered" by any car except ambulances or HEMS?

Wieso First Responder nur nl_NL? Die Chance gibt es in jedem land und kann von jedem auto außer Ambulancen oder HEMS "ausgelöst" werden?

Reply:
No, it's !==, because it's not in NL.

Reply:
Ups. Mein Fehler. Es ist schon spät ^^ . Stimmt ihr habt das ja nicht. seh ich auch grad
Oops. My bad. It's getting late ^^ . That's not true. I can see that now.